2/contrib/s2i/run

```diff
@@ -97,6 +118,8 @@ function create_jenkins_config_from_templates() {
         create_jenkins_config_xml
       elif [[ "${target_path}" == "${image_config_dir}/credentials.xml" ]]; then
         create_jenkins_credentials_xml
+      elif [[ "${target_path}" == "${image_config_dir}/proxy.yaml" ]]; then
```

Comment:
starts to be many if/elif .
Can you rewrite it using swtich/case/esac ?

Reply:
fixed